cmd/nightly/venv-tester/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/Netflix/go-expect"
+)
+
+const (
+	expectedString        = "aws-node"
+	defaultTimeoutSeconds = 30
+)
+
+func main() {
+	args := os.Args[1:3]
+
+	if len(args) != 2 {
+		printUsage()
+
+		os.Exit(1)
+	}
+
+	for _, path := range args {
+		_, err := os.Stat(path)
+		if err != nil {
+			crashPrint(err)
+		}
+	}
+
+	okctlBinaryPath := args[0]
+	clusterManifestPath := args[1]
+
+	err := testVenv(okctlBinaryPath, clusterManifestPath)
+	if err != nil {
+		crashPrint(err)
+	}
+}
+
+func testVenv(okctlBinaryPath string, clusterManifestPath string) error {
+	c, err := expect.NewConsole(expect.WithDefaultTimeout(defaultTimeoutSeconds * time.Second))
+	if err != nil {
+		return fmt.Errorf("creating pseudo console: %w", err)
+	}
+
+	defer c.Close()
+
+	cmd := exec.Command(okctlBinaryPath, "venv", "-c", clusterManifestPath)
+	cmd.Stdin = c.Tty()
+	cmd.Stdout = c.Tty()
+	cmd.Stderr = c.Tty()
+
+	log.Println("Starting venv")
+
+	err = cmd.Start()
+	if err != nil {
+		return fmt.Errorf("starting command: %w", err)
+	}
+
+	log.Println("Running kubectl -n kube-system get pods")
+	c.SendLine("kubectl -n kube-system get pods")
```

Comment:
SendLine returns an error, should we not check it? It's just a test, I'm just thinking nightly will be more difficult to debug if the error is hidden.